triagebot.toml

```
@@ -102,6 +102,15 @@ message_on_add = """\
 """
 message_on_remove = "Issue #{number}'s prioritization request has been removed."
 
+[notify-zulip."I-nominated"]
+required_labels = ["T-rustdoc"]
```

I think I might need to swap `I-nominated` and `T-rustdoc` here so that other teams could have an `I-nominated` notification in the future.

```suggestion
[notify-zulip."T-rustdoc"]
required_labels = ["I-nominated"]
```

---

+1